### PR TITLE
feat: updated styles and added delete

### PR DESCRIPTION
**Pull request by PiotrekPKP**
This PR updates the following things regarding default app's home page:
- updates post styles so that it matches the style of `create-t3-app` default home page
- adds space between the post ID and "Selected post:" text
- adds "There are no posts" message instead of showing just the scrollbar when there are not posts
- adds an option to delete posts

Screenshot from 2023-01-16 21-36-47
Screenshot from 2023-01-16 21-37-15
Screenshot from 2023-01-16 21-37-29

### CODE REVIEW

**Review by juliusmarminge — COMMENTED** [review]
Should we add a form to create posts on web too? Kinda weird right now that you can create on mobile and delete on web?

File: apps/expo/src/screens/home.tsx [hunk 1]
```diff
@@ -27,14 +27,16 @@ const CreatePost: React.FC = () => {
   const [content, onChangeContent] = React.useState("");
 
   return (
-    <View className="flex flex-col border-t-2 border-gray-500 p-4">
+    <View className="flex flex-col p-4">
       <TextInput
-        className="mb-2 rounded border-2 border-gray-500 p-2 text-white"
+        className="mb-2 rounded bg-white/10 p-2 text-white"
+        placeholderTextColor="rgba(255, 255, 255, 0.5)"
```
[the remark this answers is not in the document]
Comment (PiotrekPKP) [reply apps/expo/src/screens/home.tsx:33]: It sadly doesn't (maybe I did something wrong tho)
I'll add the form to the web version tomorrow :)

**Comment by PiotrekPKP** [comment]
Added the form and improved mobile version :+1: 
image

**Review by juliusmarminge — APPROVED** [review]
Love it! Looks great

**Pull request merged**